front/src/common-app/components/text-field.validation.ts

```diff
@@ -0,0 +1,12 @@
+import { ValidationSchema, Validators } from '@lemoncode/fonk';
+import { createFormikValidation } from '@lemoncode/fonk-formik';
+
+Validators.required.setErrorMessage('Nickname is required');
+
+const validationSchema: ValidationSchema = {
```

Comment:
This shouldn't be in common, is defined at pod level, and it should be something like

create-session.validations.ts

Is a false reusable asset, it should be defined per pod.

Reply:
Resolved on last commit